### DISCUSSION TITLE
can we get back softdeepslate in V1.21?

### DISCUSSION THREAD

**Discussion by KominoStyle** [Minecraft-AMS/Carpet-AMS-Addition#112] [Ideas]
last time i played was with version 1.20.2 and now 1.21 is out, but i also found out that the softdeepslate got removed for customhardness command, and that got removed because it was not working well since everyblock was possible to change i guess.

but could we at least get softdeepslate as a rule back? because this worked well. ♥

**Comment by 1024-byteeeee** [comment]
> last time i played was with version 1.20.2 and now 1.21 is out, but i also found out that the softdeepslate got removed for customhardness command, and that got removed because it was not working well since everyblock was possible to change i guess.
> 
> 
> 
> but could we at least get softdeepslate as a rule back? because this worked well. ♥

see #108

**Reply by KominoStyle** [reply]
but org only has chinese language, so i never used it, or is there a english one? because i would like to know how to use the mod ;)

**Reply by 1024-byteeeee** [reply]
> but org only has chinese language, so i never used it, or is there a english one? because i would like to know how to use the mod ;)

Okay, we will discuss this😉

**Reply by wendavid552** [reply]
Maybe you can try to learn some Chinese or use Machine Translation, Carpet-ORG-Addition is a very powerful carpet extension, including lots of features on e.g. fakePlayer, etc.

**Comment by 1024-byteeeee** [comment]
# Carpet-AMS-Addition v2.45.0
### Changelog
#### https://minecraft-ams.github.io/carpetamsaddition/2024/07/18/v2.45.0/

### Github
#### https://github.com/Minecraft-AMS/Carpet-AMS-Addition/releases/tag/v2.45.0

### Modrinth
#### https://modrinth.com/mod/carpet-ams-addition/versions

### CurseForge
#### https://www.curseforge.com/minecraft/mc-mods/carpet-ams-addition
